src/py42/sdk/queries/fileevents/filters/device_filter.py

```diff
@@ -2,16 +2,24 @@
 
 
 class DeviceUsername(FileEventFilterStringField):
+    """Class that filters events by Code42 username of device that observed the event."""
+
     _term = u"deviceUserName"
 
 
 class OSHostname(FileEventFilterStringField):
+    """Class that filters events by Hostname of device that observed the event."""
```

Comment:
Missing _the_ here too?

Reply:
Am I just reading it wrong? It is repeated on several filters below too.

Reply:
No, I just copy/pasted most of them and tweaked the necessary part, so some of these types of  mistakes spread everywhere.